src/Moryx.AbstractionLayer.Products.Endpoints/WorkplanController.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moryx.Products.Management.Modification;
+using Moryx.Workflows;
+using System.Collections.Generic;
+
+namespace Moryx.AbstractionLayer.Products.Endpoints
+{
+    /// Definition of a REST API on the <see cref="IWorkplansVersions"/> facade.
+    /// </summary>
+    [ApiController]
+    [Route("api/moryx/workplans/")]
+    public class WorkplanController : ControllerBase
+    {
+        private readonly IWorkplansVersions _workplansVersions;
+        public WorkplanController(IWorkplansVersions workplansVersions)
+            => _workplansVersions = workplansVersions;
+
+        [HttpGet]
+        public WorkplanModel[] GetAllWorkplans()
```

Comment:
Great, then I can remove this from workplan editing. Should also reconsider the URL.